metagraph/src/cli/stats.cpp

```diff
@@ -143,27 +166,35 @@ void print_stats(const Annotator &annotation) {
                                     / annotation.num_labels() << std::endl;
     std::cout << "representation: "
               << utils::split_string(annotation.file_extension(), ".").at(0) << std::endl;
+    using namespace annot::binmat;
```

Comment:
a new logical thing -- declaration after print, so insert an empty line
```suggestion
    std::cout << "representation: "
              << utils::split_string(annotation.file_extension(), ".").at(0) << std::endl;

    using namespace annot::binmat;
```